R/save-load_ontram.R

```diff
@@ -43,8 +42,11 @@ load_model_ontram <- function(filename, ...) {
     me <- NULL
   }
   ret <- append(rest, list(mod_baseline = mt, mod_shift = mb, mod_image = me,
-                           optimizer = tf$keras$optimizers$Adam(learning_rate = 0.001),
-                           distr = tf$sigmoid))
+                           optimizer = tf$keras$optimizers$Adam(learning_rate = rest$lr),
+                           distr = rest$distr))
   class(ret) <- "ontram"
+  if (ret$response_varying) {
+    class(ret) <- append("ontram_rv", class(ret))
```

Comment:
`append()` adds to the end of a vector, however `"ontram_rv"` should be the first class.
```r
class(ret) <- c("ontram_rv", class(ret))
```
will fix this.